gulp/tasks/scripts.js

```diff
@@ -7,6 +7,8 @@ const terser = require('@rollup/plugin-terser')
 const PluginError = require('plugin-error')
 const { rollup } = require('rollup')
 
+const MOJFrontend = require('../../src/moj/all.mjs')
+
 /**
  * Compile Sass to CSS task
```

Comment:
Is this a copy/paste error? As this function has nothing to do with sass/css - not a big deal, just confused me for a moment or two!